src/content/cv/sina-cv.md

```
@@ -0,0 +1,112 @@
+
+## Selected Projects
+
+**Axfood IT - E-Com B2B Web Team**
+  ~ 06/2025 - *Present*
+
+*Role: Backend Developer*
+ - Backend developer in Snabbgross mainly focused on the Cloud migration and modernization project.
+ -  Executed migration from Sonic MQ to AWS SQS with event-driven architecture principles. Implemented mTLS-secured communication for internal integrations. Moved from Jenkins to Bitbucket Pipelines with functional CI/CD towards SAP CCV2. 
+ - Designed and implemented performance testing strategy using Grafana k6. Established baseline performance metrics for Cloud migration project.
+ - Reduced technichal debts by aligning Spring dependencies and API definitions with updated standards.
+- Contributed to new feature development, maintenance and code review loops.
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 0

(MD005, list-indent)